src/Nri/Ui/Highlighter/V1.elm

```diff
@@ -485,13 +485,28 @@ pointerEventToActions msg model =
 
                         _ ->
                             []
+
+                focus =
```

Comment:
I thought we decided against changing the mouse behavior? or am I misunderstanding the purpose of `focus` here?

Reply:
It doesn't change the behavior, it just keeps focus on the highlightable after pressing space.